docs/source/installation.rst

```diff
@@ -100,98 +100,270 @@ Step 1: Setup dedicated python environment
+
+                    .. tabs::
+
+                        .. tab:: Linux 
+
+                            ::
+
+                                git clone https://github.com/ivadomed/ivadomed.git
+
+                                cd ivadomed
+
+                                pip install -e . --extra-index-url https://download.pytorch.org/whl/cpu
+
+                        .. tab:: Windows 
+
+                            ::
+
+                                git clone https://github.com/ivadomed/ivadomed.git
+
+                                cd ivadomed
+                                
+                                pip install -e .
 
                         .. tab:: Mac
 
-                            .. code::
+                            ::
 
-                               pip install torch==1.8.0 torchvision==0.9.0 --find-links https://download.pytorch.org/whl/torch_stable.html
+                                git clone https://github.com/ivadomed/ivadomed.git
 
+                                cd ivadomed
 
-        .. group-tab:: Repo Installation (Advanced or Developer)
+                                pip install -e .
 
-            Run this only if you have already downloaded/cloned the repo with access to the ``requirement_gpu.txt`` file, then run the following command while at the repository root level:
+                .. tab:: Contributor or Developer Installation
 
-            .. code::
+                    To contribute to the project, we recommend installing ``ivadomed`` from source along with additional dependencies related to building documentation and testing:
 
-               pip install -r requirements_gpu.txt
+                    .. tabs::
 
+                        .. tab:: Linux
+                            
+                            ::
 
+                                git clone https://github.com/ivadomed/ivadomed.git
+
+                                cd ivadomed
+
+                                pip install -e .[dev] --extra-index-url https://download.pytorch.org/whl/cpu
+
+                        .. tab:: Windows 
+
+                            ::
+
+                                git clone https://github.com/ivadomed/ivadomed.git
+
+                                cd ivadomed
+
+                                pip install -e .[dev]
+
+                        .. tab:: Mac
 
+                            ::
 
-Developer-only Installation Steps
-+++++++++++++++++++++++++++++++++
+                                git clone https://github.com/ivadomed/ivadomed.git
 
-    The additional steps below are only necessary for contributors to the ``ivadomed`` project.
+                                cd ivadomed
 
-    The ``pre-commit`` package is used to enforce a size limit on committed files. The ``requirements_dev.txt`` also contain additional dependencies related to documentation building and testing.
+                                pip install -e .[dev]
```

Comment:
The diff makes it hard to comment on the right part here, but again, I'd put all this in CONTRIBUTING.md.